src/comments.ts

```diff
@@ -26,7 +26,7 @@ export const MergeConflicted = (slug: string, user: string) => ({
 });
 
 export const ChangesRequest = (headOid: string, user: string) => ({
-    tag: `complaint-${headOid}`,
+    tag: `reviewer-complaint-${headOid}`,
     status: `@${user} One or more reviewers has requested changes. Please address their comments. I'll be back once they sign off or you've pushed new commits or comments. If you disagree with the reviewer's comments, you can "dismiss" the review using GitHub's review UI. Thank you!`
 });

```

Comment:
This is never used for looking up stuff, but at least now they are all scoped according to their domain